### ISSUE TITLE
cant start it? Installed using aur

### ISSUE THREAD

**Issue by thepenguinthatwants** [aryakaul/rusty-krab-manager#39]
I installed from aur package and gotten this error

```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/default_files.rs:55:43
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

```

**Comment by aryakaul** [comment]
Think this is related to #38, I believe the fix would me manually creating the config directory and files

```
mkdir -p ~/.config/rusty-krab-manager
```

I have an example config file in the [Wiki](https://github.com/aryakaul/rusty-krab-manager/wiki/Configuration) that you can tweak

Let me know if manually making the directory/files works, fix will be pushed with 1.2.0

**Comment by aryakaul** [comment]
Believe latest update fixes this. Please reopen if not.
